content/en/docs/setup/install/multicluster/index.md

```diff
@@ -130,7 +130,7 @@ spec:
           - registryServiceName: istio-eastwestgateway.istio-system.svc.cluster.local
             port: 15443
 EOF
-$ istioctl install --context=${CTX_CLUSTER1} -f cluster1.yaml
+$ istioctl install --context=${CTX_CLUSTER1} --skip-confirmation -f cluster1.yaml
```

Comment:
We should only add `--skip-confirmation` if we really think confirming in this task is unnecessary. If it's only to avoid it in the doc test, we should instead just pipe the input in the test, for example:
```
echo y | snip_install_istio_1
```
Also, in places where we do want to skip confirmation for simplicity, we have been trying not to highlight the fact by just adding the short version of the option at the end of the command:
```
$ istioctl install --context=${CTX_CLUSTER1} -f cluster1.yaml -y
```

Reply:
Thanks Frank. All that makes sense.